CHANGELOG.md

```diff
@@ -20,26 +20,47 @@ argument or resolve the type ambiguity by casting to the intended type.
 **Bugfixes**
 * #4619 Improve handling enum columns in compressed hypertables
```

Comment:
```suggestion
* #4619 Improve handling enum columns in compressed hypertables
```
This references the issue and not the PR. #4658 is the correct reference which is part of 2.8.1

Reply:
Good catch, entry is removed from this section